docs/index.md

```diff
@@ -37,12 +37,12 @@ To add more organizations, click the name of the current organization on the top
 
 Pulse provides push-based integrations with your workflow using the following alternatives:
 
--   **One-click integrations**
+-   **One-click integrations**<!--NOTE We might need to rethink the way we present these integrations and corresponding data/metrics-->
 
     Pulse currently supports the following "one-click" integrations that simplify the process of setting up your workflows to send data to Pulse:
 
     -   **[GitHub integration](one-click-integrations/github-integration.md) or [Bitbucket integration](one-click-integrations/bitbucket-integration.md):** reports changes, deployments, and incidents
-    -   **[PagerDuty integration](one-click-integrations/pagerduty-integration.md):** reports incidents
+    -   **[Jira integration](one-click-integrations/jira-integration.md) or [PagerDuty integration](one-click-integrations/pagerduty-integration.md):** reports incidents
```

Comment:
we already have "incidents" duplicated across git providers and Pagerduty
wouldn't it make more sense to keep the integrations unique, with just one entry for Jira?

Reply:
it's looking bery sharp! 💪